homeassistant/components/broadlink/translations/pt-BR.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Please remove all translations except the default English translation. Translations are handled by Lokalise.

Reply:
Done.